pkg/apis/wksprovider/machine/os/os.go

```diff
@@ -889,15 +900,27 @@ func createFluxSecretFromGitData(gitData GitParams, params SeedNodeParams) ([]by
 	return replaceGitFields(fluxSecretTemplate, gitParams)
 }
 
-func wksControllerManifest(controllerImageOverride, namespace, configDir string) ([]byte, error) {
+func wksControllerManifest(controllerImageOverride, ControllerImageBuiltin, namespace, configDir string) ([]byte, error) {
```

Comment:
Please pass a `ControllerParams` instead of `controllerImageOverride` and `ControllerImageBuiltin`.
In any case we don't decide to do that: `ControllerImageBuiltin` -> `controllerImageBuiltin`